spec.md

```diff
@@ -438,8 +438,8 @@ Content-Length: 0
 The rest of the upload process can be carried out with the returned url, called the "Upload URL" from the `Location` header.
 All responses to the upload url, whether sending data or getting status, will be in this format.
 
-Though the URI format (`/v2/<name>/blobs/uploads/<uuid>`) for the `Location` header is specified, clients should treat it as an opaque url and should never try to assemble it.
-While the `uuid` parameter may be an actual UUID, this proposal imposes no constraints on the format and clients should never impose any.
+Though the URI format (`/v2/<name>/blobs/uploads/<uuid>`) for the `Location` header is specified, clients SHOULD treat it as an opaque url and SHOULD never try to assemble it.
+While the `uuid` parameter MAY be an actual UUID, this proposal imposes no constraints on the format and clients SHOULD never impose any.
 
 Header `Blob-Upload-UUID` *OPTIONAL*: If clients need to correlate local upload state with remote upload state, largely for resumable uploads.
```

Comment:
ditto...

Reply:
On this one, should `SHOULD` and `MAY` be lowercase? Or just `SHOULD`?

Reply:
Ohhhh wait you want me to remove the italic and bold stuff from notations! Sorry I misunderstood.

Reply:
On this one the issue is `*OPTIONAL*` should just be OPTIONAL

Reply:
Ah it was staring me right in the face.  Done.